ui-tests/src/test/java/org/openmrs/reference/AddEditAppointmentBlockTest.java

```diff
@@ -12,13 +12,12 @@
 
 import org.junit.After;
 import org.junit.Before;
-import org.junit.Ignore;
 import org.junit.Test;
 import org.junit.experimental.categories.Category;
 import org.openmrs.reference.groups.BuildTests;
 import static org.junit.Assert.assertTrue;
 
-@Ignore
+
```

Comment:
This is good @Irenyak1  Get rid of this space as @kdaud  suggested  then all is well